src/utils/aggregator.ts

```diff
@@ -265,16 +271,18 @@ export class Aggregator {
    * @param to
    * @param feeConfig
    * @param signal
+   * @param aggregatorWaitingTime
    */
   public static async compareDex(
     baseURL: string,
     currencyAmountIn: CurrencyAmount<Currency>,
     currencyOut: Currency,
     slippageTolerance: number,
-    deadline: BigNumber | undefined,
+    deadline: number | undefined,
     to: string,
     feeConfig: FeeConfig | undefined,
-    signal?: AbortSignal,
+    signal: AbortSignal,
+    aggregatorWaitingTime: number,
```

Comment:
I think we should change name of this variable. maybe `minimumLoadingTime`

Reply:
Done.